bld/test/libtest/CMakeLists.txt

```diff
@@ -6,6 +6,15 @@
 # not want to pull in submodules for umock, etc.
 #
 
+if(WIN32)
+else()
+    check_include_file("stdint.h" HAVE_STDINT_H)
+    if(HAVE_STDINT_H)
```

Comment:
I would love to learn about under which conditions we need this, i.e. what settings you made on your system, so I can repro this on my end.